components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/IfImpl.java

```diff
@@ -457,7 +457,7 @@ protected Element getArrows(SVGDocument doc) {
                         id = prevActivity.getId() + "-" + activity.getId();
                         subGroup.appendChild(getArrowDefinition(doc, exitCoords.getXLeft(), exitCoords.getYTop(), activityEntryCoords.getXLeft(), activityEntryCoords.getYTop(), id));
                     } else {
-//                        activityEntryCoords = activity.getEntryArrowCoords();
+//                      getEndIconEntryArrowCoords  activityEntryCoords = activity.getEntryArrowCoords();
```

Comment:
The commented out lines have been removed when refactoring the code. So this is fixed.
https://github.com/wso2/carbon-business-process/blob/master/components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/IfImpl.java